tools/tools.go

```diff
@@ -24,5 +24,6 @@ import (
 	_ "github.com/ahmetb/gen-crd-api-reference-docs"
 	_ "k8s.io/code-generator/cmd/go-to-protobuf"
 	_ "k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo"
+	_ "sigs.k8s.io/controller-runtime/pkg/scheme"
```

Comment:
Why is this being added here?

Reply:
I had to add this to make the CI pass.
I think this is why: https://github.com/kubernetes-sigs/service-apis/blob/master/hack/update-proto.sh#L27. I'm not sure.